hypervisor/debug/uart16550.c

```diff
@@ -232,7 +241,7 @@ bool is_pci_dbg_uart(union pci_bdf bdf_value)
 {
 	bool ret = false;
 
-	if (uart_enabled && !serial_port_mapped) {
+	if (uart_enabled && !uart_base_address.serial_port_mapped) {
```

Comment:
use a new PR for this to merge separately.

Reply:
OK